adhocteam/Head-Start-TTADP
# Correct and bypass yarn audit vul

## Description of change
Deploys via ci/cd pipeline are blocked by new npm package vulnerabilities detected by yarn audit (our static security scan). This upgrades some npm packages to include new patches with vulnerability fixes and ignores other vulnerabilities. Ignored vulnerabilities are all found in dependent subpackages, where the main package has not been updates to use latest version, so they can not be reasonably or safely upgraded. All ignored vulnerabilities are also devDependency packages so they are unlikely to pose a security list and can be safely ignored to allow regular development to continue.


## How to test
Pipeline no longer fails, be and fe tests pass on [circleCI](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/2700/workflows/e477d6a1-1b89-4a6c-ab8d-af5bf76ec3e1)

## Issue(s)

*unticketed work


## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
